chapter_preliminaries/linear-algebra_vn.md

```diff
@@ -708,7 +708,9 @@ When we call `np.dot(A, x)` with a matrix `A` and a vector `x`, the matrix-vecto
 Note that the column dimension of `A` (its length along axis $1$) must be the same as the dimension of `x` (its length).
 -->
 
-Khi lập trình, để thực hiện nhân ma trận với vector `ndarray`, chúng ta cũng sử dụng hàm `dot` giống như tích vô hướng. Việc gọi `np.dot(A, x)` với ma trận `A` và một vector `x` sẽ thực hiện tính toán tích giữa ma trận và vector. Lưu ý rằng số chiều của cột `A` (chiều dài theo trục $1$) phải bằng với số chiều của vector `x` (chiều dài của nó).
+Khi lập trình, để thực hiện nhân ma trận với vector `ndarray`, chúng ta cũng sử dụng hàm `dot` giống như tích vô hướng. 
+Việc gọi `np.dot(A, x)` với ma trận `A` và một vector `x` sẽ thực hiện phép nhân vô hướng giữa ma trận và vector. 
+Lưu ý rằng số chiều của cột `A` (chiều dài theo trục $1$) phải bằng với số chiều của vector `x` (chiều dài của nó).
```

Comment:
```suggestion
Lưu ý rằng chiều của cột `A` (chiều dài theo trục $1$) phải bằng với chiều của vector `x` (chiều dài của nó).
```

`Số chiều` dễ nhầm với `number of dimensions`. Trong văn cảnh chỉ vector thì có thể không có nhầm lẫn, nhưng khi dùng mảng thì `số chiều` dễ bị nhầm với số chiều của mảng. Chẳng hạn, mảng một chiều là một vector, mảng hai chiều là một ma trận.